src/multiselect.js

```diff
@@ -251,6 +251,9 @@ export class Multiselect {
             block.setFieldValue(e.newValue, e.name);
           }
         });
+      } catch {
+        // Avoid errors when changing a block if it is no longer in the workspace.
+        // https://github.com/mit-cml/workspace-multiselect/issues/33
```

Comment:
We may want to consider using `console.warn()` here to let the developer know this situation has occurred.

Reply:
Updated